### ISSUE TITLE
oci_database_pluggable_database not able to delete pluggable database in state FAILED

### ISSUE THREAD

**Issue by swiftcoffee0064** [oracle/oci-ansible-collection#185]
Hello,

We have a couple of database in state FAILED ("The pluggable database was terminated at the host."). When using 
```
oci_database_pluggable_database:
    pluggable_database_id: "{{ pdb_ocid }}"
    state: absent
```
the step is marked as OK in the ansible logs but no action is done on the database.

Whereas
```
oci db pluggable-database delete --profile XXX --pluggable-database-id pdb_ocid --force
```
in the command line is working as expected on the same pluggable database.

Am I missing something?

Thanks in advance

**Comment by nikhiban** [comment]
Hi, thanks for reporting this. We are looking into this, will get back to you

**Comment by nikhiban** [comment]
Hi @swiftcoffee0064, We have fixed this issue. Please check and let us know if you face any issues

**Comment by manojmeda** [comment]
@swiftcoffee0064 The issue is fixed in the latest version (v4.3.0).

**Comment by swiftcoffee0064** [comment]
@nikhiban and @manojmeda : thanks a lot for the very quick update. It is now working as expected.
